notify.py

```diff
@@ -0,0 +1,3 @@
+import win10toast
+toaster = win10toast.ToastNotifier()
+toaster.show_toast('Rax','Its my PC',duration = 7)

```

Comment:
Can you create an exercises section to `basics/modules.md` with an exercise saying e.g. "create a program that shows a notification"? This code should then go to `basics/answers.md`.